pmm-app/src/pmm-dbaas/components/DBCluster/DBClusterParameters/DBClusterParameters.styles.ts

```diff
@@ -0,0 +1,20 @@
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+
+export const getStyles = ({
+  spacing,
+}: GrafanaTheme) => ({
+  wrapper: css`
+    display: flex;
+    flex-direction: column;
+    margin: ${spacing.xs} ${spacing.sm};
+  `,
+  connectionLoading: css`
+    display: flex;
+    justify-content: center;
+  `,
+  connectionFailed: css`
```

Comment:
And this can be renamed from connection to parameters :D

Reply:
done